backend/inspirehep/matcher/config.py

```diff
@@ -119,19 +119,6 @@
                     ],
                     "type": "nested",
                 },
-                {
```

Comment:
my bad, I was playing with the config and this was a left over. Now is the same with https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/refextract/config.py#L92